### ISSUE TITLE
Better warning(/compile error) for incorrect export file paths

### ISSUE THREAD

**Issue by st-AR-gazer** [openplanet-nl/openplanet-issues#450]
Ran into a _fun_ 'issue' that seems worth flagging for a fix. When you mess up the export file path in your toml file, the error you get makes it seem like it's a duplicate function problem (which I guess it technically is), but it isn't really clear that the issue isn't the duplicate function, but the file path in the info.toml file.

XertroV suggested we could do with a warning or a compile-time error when the system can't find an export file, and IMO that sounds like a solid idea :) . 

Basically, if there's a way to get a nudge from the compiler saying, "Hey, I can't find this file you're talking about," that would be awesome. Since I spent a while (around an hour 😢) figuring out that `Export.as != Exports.as`. This would probably save some time for someone who makes the same mistake I did. 
![image](https://github.com/openplanet-nl/issues/assets/132480072/d8252f60-ee8b-4e69-a54b-232da79f3455)
